runtime/tutor.txt

```diff
@@ -416,6 +415,296 @@ _________________________________________________________________
+   * Use ` and alt-` to set the case of selected layers to
+     upper and lower respectively.
+
+ * Type Q to record and stop recording a macro.
+   * Type q to repeat the recorded macro.
+
+
+
+
+
+
+
+
+=================================================================
+=                    5.1 USING THE JUMPLIST                     =
+=================================================================
+
+ Helix can keep track of "jumps" which are big movements, like
+ jumping to the definition of a function in code. It stores
+ these in what's called the jumplist.
+
+ Type C-s (ctrl-s) to manually save your current position to
+ the jumplist.
+
+ Type C-i and C-o to move forward and backwards in the jumplist
+ respectively.
+
+ 1. Type C-s somewhere.
+ 2. Move far away in the file.
+ 3. Type C-o (just once!) to come back to where you saved.
+
+
+
+
+
+=================================================================
+=                     5.2 SEARCHING IN FILE                     =
+=================================================================
+
+ Type / to search forward in file, enter to confirm search.
+ Type n to go to the next search match.
+ Type N to go to the previous search match.
+
+ 1. Type / and type in a common word, like 'banana'.
+ 2. Type enter to confirm the search.
+ 3. Use n and N to cycle through the matches.
+
+ Like the select command, searching also uses regex.
+
+ Note: To search backwards, type ? (shift-/).
+
+ Note: Unlike Vim, N doesn't change the search direction.
+       N always goes backwards and n always goes forwards.
+
+
+
+
+=================================================================
+=                        CHAPTER 5 RECAP                        =
+=================================================================
+
+ * Type C-s to save position to the jumplist.
+   * Type C-i and C-o to go forward and backward in the jumplist.
+
+ * Type / to search forward in file, and ? to search backwards.
+   * Use n and N to cycle through search matches.
+
+
+
+
+
+
+
+
+
+
+
+
+
+=================================================================
+=                       6.1 JOINING LINES                       =
+=================================================================
+
+ Type J to join together lines in selection.
+
+ 1. Move the cursor to the line below marked -->.
+ 2. Type x three times to select the other lines.
+ 3. Type J to join the lines together.
+
+ --> This line sentence
+is spilling over
+onto other lines.
+
+
+
+
+
+
+
+
+
+=================================================================
+=                      6.2 INDENTING LINES                      =
+=================================================================
+
+ Type > to indent a line and < to outdent it.
+
+ 1. Move the cursor to the line below marked -->.
+ 2. Indent/outdent the lines with < and >, until they are uniform.
+
+ --> These lines
+    are indented
+        very poorly.
```

Comment:
If they tried on this the output probably won't look nice given that `-->` is now indented, but should be fine for now.

Reply:
The idea was to indent the middle line and outdent the third line, leaving the `-->` line. Maybe I should make that more explicit.